src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +401,44 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
-            onValueChange={toggleChnage}
-            value={type === 'ANONYMOUS'}
+            onValueChange={toggleChange}
+            value={type === ANONYMOUS}
           />
         )}
       </View>
-      <TouchableOpacity
-        style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
-        disabled={isDisableButton()}
-        onPress={handleSendMessage}>
-        <IconSend
-          style={styles.icSendButton}
-          fillBackground={sendButtonStyle()}
-          fillIcon={colors.white}
-        />
-      </TouchableOpacity>
-      <SheetEmoji ref={refEmoji} selectEmoji={onSelectEmoji} />
+      {text?.trim() !== '' && (
+        <TouchableOpacity
+          style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
+          disabled={isDisableButton()}
+          onPress={handleSendMessage}>
+          <IconSend
+            style={styles.icSendButton}
+            fillBackground={sendButtonStyle()}
+            fillIcon={colors.white}
+          />
+        </TouchableOpacity>
+      )}
+      {text?.trim() === '' && (
+        <TouchableOpacity style={styles.btn} onPress={onSelectAttachment}>
+          <IconPlusAttachment style={styles.icSendButton} fillIcon={plusButtonStyle()} />
+        </TouchableOpacity>
+      )}
+
+      <BottomSheetAttachment
+        ref={refAttachment}
+        onOpenMedia={onOpenMedia}
+        onOpenGIF={onOpenGIF}
+        onOpenCamera={onOpenCamera}
+        onOpenFile={onOpenFile}
+        isLoadingUploadMedia={isLoadingUploadImageMedia}
+        isLoadingUploadGIF={false}
+        isLoadingUploadCamera={isLoadingUploadImageCamera}
+        isLoadingUploadFile={false}
+      />
+
+      <BottomSheetGif ref={refGif} onCancel={onCloseGIF} onSelect={handleSelectGIF} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a plus button that opens a bottom sheet for selecting attachments when there's no text in the input field. It also introduces two new bottom sheet components for handling different types of attachments. These changes seem well implemented, but make sure to test them thoroughly to ensure they work as expected in all scenarios.


<!-- This is an auto-generated comment by OSS CodeRabbit -->